webpack/JobWizard/JobWizard.js

```diff
@@ -199,8 +197,14 @@ export const JobWizard = ({ rerunData }) => {
         })
       );
     }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [rerunData, jobTemplateID, dispatch]);
+  }, [
+    rerunData,
+    jobTemplateID,
+    dispatch,
+    setDefaults,
+    setTemplateValues,
+    setAdvancedValues,
+  ]);
```

Comment:
All three functions are called within the effect and need to be tracked to prevent stale closures.